Source/Lib/Codec/EbEncHandle.c

```diff
@@ -1960,10 +1967,19 @@ void LoadDefaultBufferConfigurationSettings(
     sequenceControlSetPtr->tileGroupRowCountArray[5] = tileGroupRowCount;
 
     //#====================== Data Structures and Picture Buffers ======================
-    sequenceControlSetPtr->pictureControlSetPoolInitCount       = inputPic;
+    if (inputResolution <= INPUT_SIZE_1080p_RANGE)
+        sequenceControlSetPtr->pictureControlSetPoolInitCount       = inputPic * 2;
```

Comment:
Maybe use a constant with a meaningful name instead of just a hardcoded "2"

Reply:
Resolved.